### PR TITLE
docs: Update Publish recipes to link to community templates

### PR DESCRIPTION
**Pull request by yenly**
Keep it simple for now and when the list becomes longer, it should convert to a table on its own page.

### CODE REVIEW

**Comment by yenly** [comment]
@ingalless I don't have option to add reviewers.

**Review by riccardoferretti — APPROVED** [review]
thanks @yenly!
I agree, this needs a bit of sorting :) 

The way I think about it, there is a publishing template (e.g. foamy-gatsby), and a publishing target (e.g. vercel) - we might need better names, but that'll do for now.

Ideally the two are decoupled (I can deploy my foam-gatsby-template on vercel or whatever), atm I am not sure that's the case (I _think_ it might be, just haven't checked).

Clarifying that distinction in the docs might be helpful as well.

Happy to merge the way the PR is, but also feel free to add the above if that makes sense to you.

**Review by ingalless — APPROVED** [review]
Agree with @riccardoferretti, nice work @yenly and again thank you for creating this template!

**Comment by yenly** [comment]
@riccardoferretti I agree that way it's listed audience would need to know the distinction. Gatsby template can deploy to vercel but it requires users to run a local build. That was my experience when testing it out a few months back. I'll revisit the template and these docs and add better write-ups to this PR.

**Comment by riccardoferretti** [comment]
thanks @yenly!

**Comment by riccardoferretti** [comment]
@yenly do you want to include this distinction in this PR or wait do it in a separate one? I would like to put everything in here to wrap up the task, but what we have is already good so I don't want to hold the merge for too long if you are short in bandwidth :) let me know!

**Comment by yenly** [comment]
@riccardoferretti Thanks for the reminder! I've been distracted by Hacktober Fest issues and life. I prefer to add to this PR and look into it now.

**Comment by yenly** [comment]
@riccardoferretti @ingalless I hope the new changes add more clarity.

File: docs/recipes.md [hunk 1]
```diff
@@ -59,11 +59,15 @@ Guides, tips and strategies for getting the most out of your Foam workspace!
 
 ## Publish
 
-- Publish to [[github-pages]]
-- Publish to [[gitlab-pages]]
-- Publish your site with [[eleventy-and-netlify]]
-- Publish to [[azure-devops-wiki]]
-- Publish to [[vercel]]
+- Publish using official Foam template
```
Comment (ingalless) [inline docs/recipes.md:62]: Perhaps "publish using **an** official Foam template" would be good here? Apologies for nit-picking!

File: docs/foam-gatsby-template.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Foam Gatsby Template
```
Comment (ingalless) [inline docs/foam-gatsby-template.md:1]: Really well written, good job on this one!

**Review by ingalless — APPROVED** [review]
Great work @yenly! I've left one very picky comment that could be resolved, but it is very picky. I do apologise! Happy to approve as is though. Really good work

File: docs/recipes.md (hunk 1 above)
Comment (riccardoferretti) [reply docs/recipes.md:62]: tricky one, right now we have "one" official foam template, but I don't think it's really up to date.. but we'll get to it soon :) for now I'd say let's move forward!

**Review by riccardoferretti — APPROVED** [review]
Looks great, thanks @yenly !

**Pull request merged**